### ISSUE TITLE
"nslcd" doesn't verify system certificates

### ISSUE THREAD

**Issue by agherzan** [wheelybird/openvpn-server-ldap-otp#28]
When trying to authenticate against an LDAPS server, `nslcd` fails with:

```
nslcd: [8b4567] <authc="asd"> ldap_start_tls_s() failed (uri=ldaps://foo.bar): Can't contact LDAP server: error:14090086:SSL routines:ssl3_get_server_certificate:certificate verify failed (self signed certificate in certi...
```

There seems to be an issue with `nslcd` because testing the certificate with OpenSSL verifies the certificate as expected:

```
CONNECTED(00000003)
depth=2 O = Digital Signature Trust Co., CN = DST Root CA X3
verify return:1
depth=1 C = US, O = Let's Encrypt, CN = Let's Encrypt Authority X3
verify return:1
depth=0 CN = foo.bar
verify return:1
[...]
```

**Comment by wheelybird** [comment]
Hi.  Could you try passing the **chain.pem** certificate you get from Let'sEncrypt to LDAP_TLS_CA_CERT?   I'm not sure that CentOS has Let'sEncrypt in their trusted CA bundle file.

**Comment by agherzan** [comment]
That can't be the issue as openssl validates the certificate.

**Comment by wheelybird** [comment]
Yes, I can see that nslcd.conf isn't pointing to the system CA bundle.  I've pushed a container that should fix this:  wheelybird/openvpn-ldap-otp:cafix
Could you give it a go?  I don't have an LDAP server with a Let'sEncrypt cert I can test against right now.

**Comment by agherzan** [comment]
If I get back to it I'll test and come back with feedback. For now, I'm using wireguard,

**Comment by agherzan** [comment]
I actually tested it now. It works great, thanks. Can you reference the version which includes this change?

**Comment by wheelybird** [comment]
The fix has been included in the v1.2 release which has just been pushed to Docker Hub (wheelybird/openvpn-ldap-otp:v1.2)
